.github/scripts/build_mac.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Usage:
+#
+#     $ sh .github/scripts/build_mac.sh
```

Comment:
Eventually I'd like to run this on CI but there are complexities with setting certificated and GitHub doesn't yet support ARM, so I plan to build the releases myself for now.